content/blog/2025-05-08-go-client-GA.md

```diff
@@ -0,0 +1,501 @@
++++
+title= "Valkey GLIDE Go Client: Now Generally Available!"
+description = "Valkey GLIDE Go client reaches general availability. Read to learn more about the go client designed for performance and developer productivity"
+date= 2025-05-08 01:01:01
+authors= [ "niharikabhavaraju", "jbrinkman"]
++++
+
+The Valkey GLIDE contributors are pleased to announce the general availability (GA) release of the GLIDE (General Language Independent Driver for the Enterprise) Go client. This release brings the power and reliability of Valkey to Go developers with an API designed for performance and developer productivity.
```

Comment:
proposal
```suggestion
The Valkey GLIDE contributors are pleased to announce the general availability (GA) release of the Valkey GLIDE Go client. This release brings the power and reliability of Valkey to Go developers with an API designed for performance and developer productivity.
```

\+ you can add repo and/or pkg link

Reply:
Next line describes 2.0 version, like 1.x was skipped. Maybe need to reword or add more details?